saleor/graphql/channel/mutations/channel_create.py

```diff
@@ -89,6 +114,13 @@ def clean_input(cls, info, instance, data, input_cls=None):
             cleaned_input["slug"] = slugify(slug)
         if stock_settings := cleaned_input.get("stock_settings"):
             cleaned_input["allocation_strategy"] = stock_settings["allocation_strategy"]
+        if order_settings := cleaned_input.get("order_settings"):
```

Comment:
Calling update only for this field should be allowed with `MANAGE_ORDERS` permissions.